### PR TITLE
[SD-3913] - Keywords facets in search

### PR DESCRIPTION
**Pull request by akintolga**

### CODE REVIEW

File: client/app/scripts/superdesk-search/views/search-facets.html [hunk 1]
```diff
@@ -91,6 +91,15 @@
             </ul>
         </div>
 
+        <div sd-toggle-box data-title="{{ 'Keywords' | translate }}" ng-hide="isEmpty('keywords')" data-open="true">
```
Comment (sivakuna-aap) [inline client/app/scripts/superdesk-search/views/search-facets.html]: One time binding?

**Comment by sivakuna-aap** [comment]
:+1: except the minor one

**Comment by akintolga** [comment]
Depends on https://github.com/superdesk/superdesk-core/pull/131

**Comment by ioanpocol** [comment]
Core PR  superdesk/superdesk-core#131 is merged. Please update the requirements file.

**Comment by mugurrus** [comment]
:+1: (requirements file update before merge)

**Pull request merged**